packages/block-library/src/page-list/block.json

```diff
@@ -51,6 +51,18 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

Comment:
These border controls should probably not be displayed by default given the block doesn't already have spacing support controls shown by default.

Reply:
```suggestion
```

These should be hidden by default given the block didn't already have spacing support and is getting it primarily for border-radius support and consistency.